services/simcore/docker-compose.yml

```diff
@@ -858,10 +857,3 @@ networks:
   interactive_services_subnet:
     name: ${SWARM_STACK_NAME}_interactive_services_subnet
     external: true
-# self-signed only
-secrets:
```

Comment:
Might it have been used in `local` deployments?

Reply:
yes, but I am not sure why it was in this file specifically. Since these lines, that are in fact needed for the local deployment, are also written in `docker-compose.local.yml` when I checked